plugins/auth/pyproject.toml

```diff
@@ -6,9 +6,16 @@ build-backend = "hatchling.build"
 name = "fps_auth"
 description = "An FPS plugin for the authentication API"
 keywords = [ "jupyter", "server", "fastapi", "pluggy", "plugins",]
+dynamic = ["version"]
 requires-python = ">=3.7"
-dependencies = [ "fps[uvicorn] >=0.0.17", "fps-lab", "fps-login", "aiosqlite", "fastapi-users[sqlalchemy,oauth] >=10.1.4,<11",]
-dynamic = [ "version",]
+dependencies = [
+	"fps[uvicorn] >=0.0.17",
+	"fps-lab",
+	"fps-login",
+	"aiosqlite",
+	"fastapi-users[sqlalchemy,oauth] >=10.1.4,<11"
```

Comment:
You didn't take my previous suggestion into account (no tab).
It's also fine not to commit this change.